common/sgx/iohandler/file_io.cpp

```diff
@@ -0,0 +1,151 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include <stdlib.h>
+#include <stdio.h>
+#include <iostream>
+#include <stdint.h>
+
+#include "file_io.h"
+#include "iohandler_enclave.h"
+#include "enclave_utils.h"
+
+// Maximum size of buffer that is used for file io
```

Comment:
Please avoid global functions. Implement functions inside C++ class and namespace.

Reply:
Done